src/04_printing.py

```diff
@@ -14,4 +14,5 @@
 
 # Use the 'format' string method to print the same thing
```

Comment:
Hey Ramy, great job!

Remember to finish all of the instructions though.

Reply:
it's all done now , thanks Kavin